app/src/main/AndroidManifest.xml

```diff
@@ -246,6 +246,7 @@
                 <!-- video prefix -->
                 <data android:pathPrefix="/embed/"/>
                 <data android:pathPrefix="/watch"/>
+                <data android:pathPrefix="/"/>
```

Comment:
If you're doing that, you should remove the 2 lines above, since they also start with a / ;)

Reply:
and also the lines below. thank you